addons/base_vat/models/base_vat.py

```diff
@@ -89,18 +70,19 @@ def simple_vat_check(self, cr, uid, country_code, vat_number, context=None):
             return False
         check_func_name = 'check_vat_' + country_code
         check_func = getattr(self, check_func_name, None) or \
-                        getattr(vatnumber, check_func_name, None)
+            getattr(vatnumber, check_func_name, None)
         if not check_func:
             # No VAT validation available, default to check that the country code exists
             if country_code.upper() == 'EU':
                 # Foreign companies that trade with non-enterprises in the EU
                 # may have a VATIN starting with "EU" instead of a country code.
                 return True
-            res_country = self.pool.get('res.country')
-            return bool(res_country.search(cr, uid, [('code', '=ilike', country_code)], context=context))
+            ResCountry = self.env['res.country']
+            return bool(ResCountry.search([('code', '=ilike', country_code)]))
```

Comment:
Maybe add limit=1 to avoid bringing in all of the database, and use search_count so it actually brings no data, just 0 or 1?